_posts/2020-06-30-not-only-the-code-benefits-from-the-open-source-community.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Not Only The Code Benefits From The Open Source Community
+date: 2020-06-30  9:08:11
+author: Sergei Petrosian
+
+tags:
+- foreman
+- katello
+- documentation
+---
+
+The upstream versions of the Foreman and Katello projects become among other projects like Atix's Orcarhino, the Red Hat Satellite of tomorrow. The discussions and contributions that take place in the vibrant upstream community shape the Red Hat Satellite code base. The code base for Satellite begins upstream and moves downstream. Until now, the Satellite documentation did not follow the same journey.
```

Comment:
Few comments here: 

Replace: 

`The upstream versions of the Foreman and Katello projects become among other projects like Atix's Orcarhino, the Red Hat Satellite of tomorrow.`

with 

`The upstream versions of the Foreman and Katello projects become, among other projects, the ATIX Orcharhino and the Red Hat Satellite of tomorrow.`

And:

`The discussions and contributions that take place in the vibrant upstream community shape the code base. The code base begins upstream and moves downstream. Until now, the Satellite documentation did not follow the same journey. In this blog, I will outline what has been happening with Satellite documentation over the last year and how this benefits both the Foreman community and Red Hat Satellite users.`

Reply:
Although this has been slightly changed, this comment still stands.